shell/platform/android/io/flutter/embedding/engine/FlutterShellArgs.java

```diff
@@ -47,6 +47,8 @@
   public static final String ARG_VERBOSE_LOGGING = "--verbose-logging";
   public static final String ARG_KEY_OBSERVATORY_PORT = "observatory-port";
   public static final String ARG_OBSERVATORY_PORT = "--observatory-port=";
+  public static final String ARG_KEY_WRITE_SERVICE_INFO = "write-service-info";
```

Comment:
Shouldn't this also get wired up in the old embedding class?

Reply:
TIL we have more than one embedding

Reply:
Also nit: this should have documentation.

https://google.github.io/styleguide/javaguide.html#s7.3-javadoc-where-required